examples/tests/sdk_tests/std/range.test.w

```diff
@@ -1,2 +1,52 @@
 //-----------------------------------------------------------------------------
-// TODO: https://github.com/winglang/wing/issues/2785
\ No newline at end of file
+// rangeOf
+
+assert(std.Range.of(1,5,true).join() == "1,2,3,4,5");
+
+test "rangeOf" {
+    assert(std.Range.of(1,5,true).join() == "1,2,3,4,5");
+}
+
+assert(std.Range.of(1,5,false).join() == "1,2,3,4");
+
+test "rangeOfWithoutLastNumber" {
+   assert(std.Range.of(1,5,false).join() == "1,2,3,4");
+}
+
+assert(std.Range.of(1,5).join() == "1,2,3,4");
+
+test "rangeOfWithoutLastNumberDefault" {
+   assert(std.Range.of(1,5).join() == "1,2,3,4");
+}
+
+assert(std.Range.of(-3,3,true).join() == "-3,-2,-1,0,1,2,3");
+
+test "rangeOfWithNegativeRange" {
+   assert(std.Range.of(-3,3,true).join() == "-3,-2,-1,0,1,2,3");
+}
```

Comment:
Maybe one more with a negative range and false at the last argument? (as well as decreasing one with false at the last argument)